tests/in-person-payments/in-person-payments.spec.js

```diff
@@ -0,0 +1,64 @@
+﻿const { test, expect } = require('@playwright/test');
+
+// In Person Payments
+test('In-person Payments', async ({ browser }) => {
+    const context = await browser.newContext();
+    const page = await context.newPage();
+
+    /* 1. Payment Request */
+    await page.goto('/');
+    
+    await expect(page).toHaveTitle(/In-person Payments Demo/);
+
+    await page.getByRole('heading', { name: /Adyen In-person Payment Demo/ })
+    await page.getByRole('link', { name: 'Click here to settle the bill for a restaurant' }).click();
+
+    await page.getByText(/Table 1/).click();
+    await page.getByRole('button', { name: 'Pay' }).click();
+    
+    // Mock terminal
+    await handleMockTerminalPage(context);
+
+    // Return to cash register page
+    await page.getByRole('link', { name: 'Return' }).click(); // Return to cash register page
+    
+    /* 2. TransactionStatus Request */
+    await page.getByText('Table 1').click();
+    await page.getByRole('button', { name: 'Transaction Status' }).click();
+    await page.getByRole('link', { name: 'Return' }).click(); // Return to cash register page
+
+
+    /* 3. Reversal Request */
+    await page.getByText('Table 1').click();
+    await page.getByRole('button', { name: 'Reversal' }).click();
```

Comment:
Coud add an expectation to verify the Reversal click has worked as expected?

Reply:
If we're unable to find the button, that already tells us that it won't work. If it's not clickable, it will get stuck there and fail the test. I think this is sufficient for now